tests/MongoDB.Driver.Tests/AsyncCursorTests.cs

```diff
@@ -65,4 +116,10 @@ private void DropCollection(IMongoClient client, string databaseName, string col
             client.GetDatabase(databaseName).DropCollection(collectionName);
         }
     }
+
+    public static class AsyncCursorReflector
+    {
+        public static long _cursorId(this AsyncCursor<BsonDocument> obj) =>
+            (long) Reflector.GetFieldValue(obj, nameof(_cursorId));
+    }
```

Comment:
Minor optional nit: since we're in already in this file, I'd update it so that it ends with a new line per POSIX standards :)

Reply:
Done!